src/stubs.ts

```diff
@@ -21,7 +22,7 @@ import { Stub, Stubs } from './types'
 
 interface StubOptions {
   name: string
-  propsDeclaration?: ComponentPropsOptions
+  propsDeclaration: ComponentPropsOptions
```

Comment:
is this necessary? Can't that be left optional, and have a default value in the stubs?

Reply:
Done, fixed all comments, thanks for the review!